docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3118,9 +3119,33 @@ error rate is not very intuitive for the end user.
 At some point in the future, we might make this more user friendly. However, for the 
 sake of a minimum functional model, this is satisfactory. 
 
-One function has an additional parameter. The typographic function needs an input for 
-the probability that a corrupted token contains the original token. For simplicity, 
-we will use 0.1 and the user will not be able to edit this value. 
+A few functions have additional parameters that allow the user more control. These 
+inputs will not be specified in the main settings for pseudopeople listed above. We 
+imagine that a yaml file (or similar file type) will be available for users to download 
+and adjust the additional inputs if desired. The table below outlines these inputs, 
+their purpose, and the default value for the yaml file. 
+
+.. list-table:: Additional Inputs and Default Values
+  :widths: 20 20 20 20
+  :header-rows: 0
+
+  * - Noise Function Affected
+    - Observer Data Affected 
```

Comment:
Are these inputs being set for _each_ column they are affecting, or is there one value for all columns affected by that noise function?

Reply:
Currently, parameters are at the column specific level. So you can specify for typographic errors that the rate for first names is higher than for street name.